admin/lib/solidus_admin/configuration.rb

```diff
@@ -59,6 +59,19 @@ class Configuration < Spree::Preferences::Configuration
     preference :importmap_paths, :array, default: [
       SolidusAdmin::Engine.root.join("config", "importmap.rb"),
     ]
+
+    # The method used to authenticate the user in the admin interface, it's expected to redirect the user to the login method
+    # in case the authentication fails.
+    preference :authentication_method, :string, default: :authenticate_solidus_backend_user!
+
+    # The method used to retrieve the current user in the admin interface.
+    preference :current_user_method, :string, default: :spree_current_user
+
+    # The path used to logout the user in the admin interface.
+    preference :logout_link_path, :string, default: :destroy_solidus_user_session_path
+
+    # The HTTP method used to logout the user in the admin interface.
+    preference :logout_link_method, :string, default: :delete
```

Comment:
I like the idea of having the destroy session path and the action to use for it here 👍
(even if they do not seem to be used at the moment 👀 )

About the logout_link_method, this seems similar to the sign_out_via option on Devise:
https://github.com/heartcombo/devise/blob/ec0674523e7909579a5a008f16fb9fe0c3a71712/test/rails_app/config/initializers/devise.rb#L172-L173